### ISSUE TITLE
Closed issues but no release

### ISSUE THREAD

**Issue by hshuaib90** [GSTT-CSC/hazen#120]
Come across an issue where a ticket has been closed and the fix has been merged into develop but then there's been no release, which means the user re-raises the issue.

Need to figure out a system to avoid this from happening.

**Comment by hshuaib90** [comment]
I think the best solution is to create a Project board for the next few releases (or one at least). And once the backlog for a release is done, we need to create a release branch and update and merge with master and develop.

There should also be a release manager for each release.

**Comment by laurencejackson** [comment]
Sorry @hshuaib90 just saw this. I think this all seems sensible and is what we have been doing lately? Also agree that a release manager to handle each release would be a useful way to spread the workload, happy to help with this.

I would also lean towards a regular release cycle as opposed to a release whenever a backlog is complete since a particularly difficult issue/PR might lead to delays and we'll have the same problem that you mention above.

**Comment by hshuaib90** [comment]
I think it's helpful to plan and prioritize what is coming in a release. If thing's are taking too long they can always be bumped to the next release.

**Comment by hshuaib90** [comment]
Closed with #147
